pkg/bufferdecoder/net_decoder.go

```diff
@@ -118,3 +120,80 @@ func (decoder *EbpfDecoder) DecodeNetDebugEvent(netDebugEvent *NetDebugEvent) er
 	decoder.cursor += int(netDebugEvent.GetSizeBytes())
 	return nil
 }
+
+// getDnsLayerFromBytes creates a packet from packetBytes and returns DNS layer
+func getDnsLayerFromBytes(packetBytes []byte) (*layers.DNS, error) {
```

Comment:
i think `getDnsLayerFromBytes` is ok here because it returns `*layers.DNS` - which is the DNS layer of the packet.
in my opinion `common` suggests that its something we defined and use for different types of DNS packets.

in any case, i don't dell strognly about it - so tell me if you think i should change it.

Reply:
I think its okay if you prefer the other name. Resolving this.